modules.d/95udev-rules/module-setup.sh

```diff
@@ -57,9 +56,21 @@ install() {
     # eudev rules
     inst_rules 80-drivers-modprobe.rules
 
-    if dracut_module_included "systemd"; then
-        inst_multiple -o ${systemdutildir}/network/*.link
-        [[ $hostonly ]] && inst_multiple -H -o /etc/systemd/network/*.link
```

Comment:
Speaking without knowing all our customer use cases, wouldn't this break any setup with predictable naming and custom .link files? Shouldn't we replace line 72 with lines 61 and 62?